packages/react-theme-sass/src/index.ts

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
Is the `index.ts` necessary?

Reply:
no, you should also remove the main entrypoint